spec/board_spec.rb

```diff
@@ -60,4 +60,149 @@
 			expect(@new_board.open_row).to eq([6,5,6,6,6,6,6])
 		end
 	end
+
+	describe '#check_win?' do
+    before(:each) do
+      @new_board = Board.new
+    end
+
+    it 'returns false if there is no win' do
+      @new_board.grid = {
+        row_0: "ABCDEFG",
+        row_1: ".......",
+        row_2: ".......",
+        row_3: ".......",
+        row_4: ".......",
+        row_5: ".OXO...",
+        row_6: ".XOXX..",
+        }
+        expect(@new_board.check_win?("X", 1)).to be False
+    end
+
+		it 'returns false if there is no win' do
+      @new_board.grid = {
+        row_0: "ABCDEFG",
+        row_1: ".......",
+        row_2: ".......",
+        row_3: "X...XOX",
+        row_4: "OXOXOXO",
+        row_5: "OXOXOOO",
+        row_6: "XOXOXOX",
+        }
+
+        expect(@new_board.check_win?("O",1)).to be False
```

Comment:
Great. lets get started with these tests and we can update the rest to pass more arguments later